docs/development/workflow/git_edit_workflow_examples.rst

```diff
@@ -52,98 +43,101 @@ Grab the latest updates to astropy
 A few steps in this tutorial take only a single command. They are broken out
 separately to outline the process in words as well as code.
 
-Inform your local copy of Astropy about the latest changes in the development
+Inform your local copy of ``astropy`` about the latest changes in the development
 version with::
 
     git fetch astropy --tags
 
 Set up an isolated workspace
 ============================
+
 + Make a new `git`_ branch for fixing this issue, check it out, and let my
   GitHub account know about this branch::
 
-    git branch fix-1761 astropy/master   # branch based on latest from GitHub
-    git checkout fix-1761                   # switch to this branch
-    git push --set-upstream origin fix-1761 # tell my github acct about it
+    git checkout astropy/master -b fix-1761 # branch based on latest from GitHub
+                                            # and switch to this branch
+    git push your-user-name fix-1761        # tell my GitHub acct about it
 
-+ Make a python environment just for this fix and switch to that environment.
-  The example below shows the necessary steps in the Anaconda python
++ Make a Python environment just for this fix and switch to that environment.
+  The example below shows the necessary steps in the Miniconda/Anaconda Python
   distribution::
 
-        conda create -n apy-1761 --clone root  # Anaconda distribution only
-        source activate apy-1761
+    conda create -n apy-1761 python=3.9 # replace 3.9 with desired version
```

Comment:
Since we now use `tox` for testing, which sets up its own environment anyway, I think we should remove this as unnecessary complication...

Reply:
Given #10851 , I am hesitant to introduce `tox` here. Can we do the `tox` stuff as a follow-up PR and let someone who really likes `tox` do it? (I don't use `tox`.)

Reply:
OK, but doesn't one somewhere actually need to do `setup.py develop`, so that the C stuff is properly compiled? Or are we relying on auto-compilation here?

Reply:
`pip install -e .` has always worked for me. It also compiles the C stuff. I never do `setup.py develop` myself.